### PR TITLE
Delay creating the global JRuby instance to org.embulk.EmbulkRunner.

### PR DESCRIPTION
**Pull request by dmikurube**
@muga Yay, we're close to the last step! Moving the creation of the singleton JRuby instance to `EmbulkRunner` in this PR.

It assumes #755 is merged beforehand.

After this PR, the real last step is to move the initial JRuby configuration code to `JRubyScriptingModule`. There are many barriers, though. (E.g. we need to take care of `EmbulkEmbed` users, we need a way to store the configuration values somewhere to pass `JRubyScriptingModule`, ...)

### CODE REVIEW

File: embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java [hunk 1]
```diff
@@ -114,6 +122,27 @@ public static ScriptingContainer setup(
 
     private static final class UnrecognizedJRubyOptionException extends Exception {}
     private static final class NotWorkingJRubyOptionException extends Exception {}
+    private static final class UnrecognizedJRubyLoadPathException extends Exception {
+        public UnrecognizedJRubyLoadPathException()
+        {
+            super();
+        }
```
Comment (muga) [inline embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java]: minor: don't you need to declare an unused constructor?

File: embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java [hunk 2]
```diff
@@ -114,6 +122,27 @@ public static ScriptingContainer setup(
...
+
+        public UnrecognizedJRubyLoadPathException(final String message)
+        {
+            super(message);
+        }
+
+        public UnrecognizedJRubyLoadPathException(final Throwable cause)
+        {
+            super(cause);
+        }
```
Comment (muga) [inline embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java]: minor: this is also not used.

File: embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java (hunk 1 above)
Comment (dmikurube) [inline embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java]: Exactly. Removed.

File: embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java (hunk 2 above)
Comment (dmikurube) [inline embulk-core/src/main/java/org/embulk/EmbulkGlobalJRubyScriptingContainer.java]: Ditto. Thanks.

**Comment by dmikurube** [comment]
Merging!

**Pull request merged**